api/envoy/config/route/v3/route_components.proto

```diff
@@ -210,6 +210,18 @@ message VirtualHost {
   // value of this and the listener per_connection_buffer_limit_bytes.
   google.protobuf.UInt32Value per_request_buffer_limit_bytes = 18;
 
+  // The maximum bytes which will be buffered for request bodies to support large request body
+  // buffering beyond the ``per_connection_buffer_limit_bytes``.
+  //
+  // This limit is specifically for request body buffering and allows buffering larger inference
```

Comment:
what is meant by inference payloads here?

Reply:
It refers to the Model Serving/Inference workloads which are typically large. See the Background section [here](https://github.com/envoyproxy/envoy/issues/40028). If it's confusing then I'm happy to change it to what you prefer.

Reply:
Since this is general API (not specific to ML), perhaps just saying "larger payloads while maintaining..." would be better (i.e. remove "inference")?  In any case, I'm fine either way.  Thanks!